### DISCUSSION TITLE
"sudo so-rule"-command disabled?

### DISCUSSION THREAD

**Discussion by davidjud** [Security-Onion-Solutions/securityonion#12168] [2.4] [answered]
### Version

2.4.30

### Installation Method

Network installation on Ubuntu

### Description

other (please provide detail below)

### Installation Type

Import

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

20

### Storage for /

500

### Storage for /nsm

100

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I tried to disable some NIDS rules in the opt/so/rules directory, just like a tutorial-video from the offical SO youtube account shows. But I found out that the so-rule command was removed with the 2.4 Version. 
My question now is if there is any other possibility of disabling any rules via the command line.
Thanks in advance.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment, marked as answer]
For disabling rules use https://docs.securityonion.net/en/2.4/managing-alerts.html#disable-the-alert
